website/content/maintainers/security/security-guidelines.md

```diff
@@ -0,0 +1,241 @@
+
+It is critical to ensure no sensitive information is exposed as part of the source code, the documentation or any configuration in the source code repository. It may sometimes accidentally escape even the vigilant eyes of the contributor and reviewers, hence we recommend automating this activity and implementing secret scanning as part of the continuous integration process in the source code repository.
+
+There are several tools and projects aimed at providing secret scanning services, including but not limited to the ones mentioned in the section "Develop" of the Cloud Native Security Map[[3]]. GitHub also provides its own secret scanning service, you can find more details of this service at [Keeping secrets secure with secret scanning - GitHub Docs](https://docs.github.com/en/code-security/secret-scanning)
+
+### Code scanning (recommended)
+
+Code scanning is an automated security test to identify vulnerabilities and errors in the source code without actually compiling or executing the code. This is a key testing strategy to shift security testing left and we recommend configuring code scanning and utilizing its insights in your projects. There are several tools (both commercial and open source) available to perform the code scanning or an array of languages and technologies. GitHub also provides code scanning functionalities, which can be seen in their documentation - [Automatically scanning your code for vulnerabilities and errors - GitHub Docs](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors).
+
+In addition to scanning your code for identifying vulnerabilities and errors, managing the vulnerabilities dependencies is also important. [Dependabot](https://github.com/dependabot) is one tool that helps in managing vulnerabilities in dependencies by automatically raising a pull request to update vulnerable versions to secure versions of that dependency. These pull requests can be then analyzed, and further action can be taken. Further details of Dependabot and configuring it in your project code repository can be found at [Automatically updating dependencies with known vulnerabilities with Dependabot security updates - GitHub Docs](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates).
+
+For further information on securing the code repository, we recommend reviewing the **GitOps section** of the CNCF Cloud Native Security Whitepaper - [tag-security/cloud-native-security-whitepaper.md at main · cncf/tag-security (github.com)](https://github.com/cncf/tag-security/blob/main/security-whitepaper/v2/cloud-native-security-whitepaper.md#gitopsnew-in-v2) and the **Control Environments** sections of the Software Supply Chain Best Practices paper [tag-security/CNCF\_SSCP\_v1.pdf at main · cncf/tag-security (github.com)](https://github.com/cncf/tag-security/blob/main/supply-chain-security/supply-chain-security-paper/CNCF_SSCP_v1.pdf)
```

Comment:
I recommend updating any links that are formatted like this, where the link text is not a title but instead the path in GitHub, to use a more descriptive link text that properly hints to the user where the content will go.

That better matches the style used on the contribute.cncf.io site and is more accessible for people using screen readers.

Reply:
Addressed in commit [4924939](https://github.com/cncf/tag-contributor-strategy/pull/362/commits/4924939f7c089717b2abc3d461fb867424049fc1)
Please let me know if you need any additional changes. Thanks